lib/arrays.md

```diff
@@ -5,44 +5,65 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is a list of items.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+Team rosters would work well in an array, as would an ingredient list.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+If your array is simply
+Days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+#you can simply
+Days[whatver day you want to call]
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+Sure, with this method, Ruby would count down, starting at 0 then negative, backwards around the array until the right one.
 
-# How would you perform an operation on every element inside an Array?
+# ?How would you perform an operation on everey lement inside an Arrayy 
 
-Replace me with your response...
+I am really struggling with this.... am I trying to add bullets to a list of names?  or make a list of numbers to perform mathmatical opperations on?
```

Comment:
Alright, after way too much research I found what I was missing to loop a string or array!!
scores = ['32','45','12']
scores.each do |scar|
    puts 'Score: '+scar
end

It does make a huge difference whether the numbers are as a string or an array, so if they are in the quotes or not is vital.  So the first step is to convert the array of numbers into a string.  The .join command can convert the array into a string easily, then the loop with the 'scores: ' Title.